docs/docs/airgap.md

```diff
@@ -72,7 +72,7 @@ After downloading the dedicated air-gap XOA provided by our support team, follow
 1. **Obtain the Deployment Script**: On a machine with internet access (or any non-air-gap machine), run the following command to download the deployment script:
 
    ```bash
-   curl https://xoa.io/deploy > deploy.sh
+   wget -qO deploy.sh https://xoa.io/deploy
```

Comment:
We need to check that it works with all supported versions: https://docs.xen-orchestra.com/supported_hosts

Reply:
`--no-verbose` is better because it does not hide error messages.

```suggestion
   wget --no-verbose -O deploy.sh https://xoa.io/deploy
```